ZFSin/zfs/lib/libzfs/libzfs_import.c

```diff
@@ -1560,7 +1563,7 @@ zpool_open_func_win(void *arg)
 		* Try to read the disk label first so we don't have to
 		* open a bunch of minor nodes that can't have a zpool.
 		*/
-		check_slices(rn->rn_avl, fd, rn->rn_name);
+		check_slices(rn->rn_avl, HTOI(fd), rn->rn_name);
```

Comment:
Uses direct `HTOI()` macro as this is `zpool_open_func_win()` - ie the `_win()` version of the whole function.